markdown/exercises/01-workshop_exercises.md

```diff
@@ -453,15 +453,15 @@ at `x=y=0` and see a periodic orbit, e.g., `ts=0:0.05:22; plot(ts, sol1.(ts,
 idxs=1))`.
 
 If you are not familiar with this process, see
-[the Gierer-Meinhardt example from the SciMLTutorials.](http://juliadiffeq.org/SciMLTutorials.jl/html/introduction/03-optimizing_diffeq_code.html)
+[the Gierer-Meinhardt example from the SciMLTutorials.](http://tutorials.juliadiffeq.org/html/introduction/03-optimizing_diffeq_code.html)
 
 Note: Start by doing the simplest implementation!
 
 ## Part 2: Optimizing the BRUSS Code
 
 PDEs are expensive to solve, and so we will go nowhere without some code
 optimizing! Follow the steps described in the
-[the Gierer-Meinhardt example from the SciMLTutorials](http://juliadiffeq.org/SciMLTutorials.jl/html/introduction/03-optimizing_diffeq_code.html)
+[the Gierer-Meinhardt example from the SciMLTutorials](http://tutorials.juliadiffeq.org/html/introduction/03-optimizing_diffeq_code.html)
```

Comment:
should be tutorials.sciml.ai

Reply:
I'm doing that change everywhere in a separate PR.